### PR TITLE
[0.9.x] KOGITO-1120 - Create simple 'greeting' example (kogito + serverless w…

### PR DESCRIPTION
**Pull request by tsurdilo**
…… (#170)

* KOGITO-1120 - Create simple 'greeting' example (kogito + serverless workflow)

* fixed typo

* Updated examples to latest serverless runtime

* updated example to also add switch state

* fixing spanish spelling

* Renamed according to latest master changes

### CODE REVIEW

**Comment by tsurdilo** [comment]
depends on https://github.com/kiegroup/kogito-runtimes/pull/435

**Review by cristianonicolai — APPROVED** [review]

File: serverless-workflow-greeting-quarkus/pom.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+<project
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-examples</artifactId>
+    <version>8.0.0-SNAPSHOT</version>
```
Comment (radtriste) [inline serverless-workflow-greeting-quarkus/pom.xml]: please update to 0.9.1-SNAPSHOT

**Review by radtriste — CHANGES_REQUESTED** [review]
small change to make and then it should be good

File: serverless-workflow-greeting-quarkus/pom.xml (hunk 1 above)
Comment (tsurdilo) [reply serverless-workflow-greeting-quarkus/pom.xml]: updated - nice catch!

**Review by radtriste — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
This one is ready @cristianonicolai

**Pull request merged**